tedana/workflows/tedana.py

```diff
@@ -677,13 +706,17 @@ def tedana_workflow(
             )
             ica_selector = selection.automatic_selection(comptable, n_echos, n_vols, tree=tree)
             n_likely_bold_comps = ica_selector.n_likely_bold_comps
-            if (n_restarts < maxrestart) and (n_likely_bold_comps == 0):
-                LGR.warning("No BOLD components found. Re-attempting ICA.")
-            elif n_likely_bold_comps == 0:
-                LGR.warning("No BOLD components found, but maximum number of restarts reached.")
-                keep_restarting = False
-            else:
+
+            if ica_method=='robustica':  #########BTBTBT
```

Comment:
Stick to double-quotes for strings if that's what the code base uses

Reply:
Fixed.